### ISSUE TITLE
Inability to zoom on stereoplot

### ISSUE THREAD

**Issue by din14970** [pyxem/orix#165]
I noticed that with the new implementation of the stereographic plots it has become impossible to zoom in on the data or otherwise change the axes limits. This is a bit annoying in my case where I really only care about a small portion of the stereogram: 
![afbeelding](https://user-images.githubusercontent.com/25320842/110338672-9497fa80-8027-11eb-8cb5-b64f33347e7e.png)

Do you think it's possible to make zooming possible again?

**Comment by hakonanes** [comment]
Definitely! Whether we can zoom or not is controlled here: https://github.com/pyxem/orix/blob/master/orix/plot/stereographic_plot.py#L218. I'll set up a PR right away.

**Comment by hakonanes** [comment]
Note that I plan to enable restricting the spines (plottable domain) by using three vectors (say 001, 101, 111), similar to how it is done for Matplotlib's polar plot: https://matplotlib.org/stable/gallery/pie_and_polar_charts/polar_scatter.html#scatter-plot-on-polar-axis-confined-to-a-sector. This will enable us to easily plot inverse pole figures, etc. But first we need crystallographically aware vectors (https://github.com/pyxem/orix/pull/164).

**Comment by hakonanes** [comment]
Actually, turns out to be more difficult than initially anticipated. Sorry. Solution for now is to use the stereographic plot you made before I changed it: https://github.com/pyxem/orix/commit/dc2f2aad5b12bdebe0c2fa2468afe1b289b8acb8.

This will move the ability to plot only parts of the pole figure up on my agenda.

**Comment by hakonanes** [comment]
This is now possible in #232, which enables IPF functionality which will be added in a follow-up PR.
